src/main/kotlin/menu/ArchiveMenu.kt

```diff
@@ -0,0 +1,24 @@
+package menu
+
+import model.Archive
+import model.menu.Menu
+import repository.Repository
+import java.util.Scanner
+
+class ArchiveMenu(archive: Archive) {
+    private val menu = Menu("\uD83D\uDDC4\uFE0F Архив ${archive.name}: ")
+    private val addNote: () -> Unit = { Repository.addNote(archive) }
+    private val getNotes: () -> Unit = { Repository.getNotes(archive) }
+    private val back : () -> Unit = { }
```

Comment:
```suggestion
    private val back : () -> Unit = Unit
```
Можно так написать, чтобы не создавать лямбду, а использовать синглтон объект Unit